src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -197,11 +197,11 @@ private function finishRequest(Request $request, int $type)
     }
 
     /**
-     * Handles an exception by trying to convert it to a Response.
+     * Handles a throwable by trying to convert it to a Response.
      *
      * @throws \Exception
```

Comment:
this can be removed as it is wrong and useless. it could also throw a throwable if an event listener throws a throwable

Reply:
I think what this `@throws \Exception` means is that line 215 the method can throw `$event->getException()` that is an `\Exception`.

Reply:
Still think this annotation is pointless. Why don't you rethrow `$event->getThrowable()`?

Reply:
> Why don't you rethrow $event->getThrowable()
To respect the annotation 😅 
Let's remove this annotation.